{"uid":"26d27b889b74cde1","name":"test_removal_of_non_primary_shard","fullName":"tests.integration.sharding_tests.test_sharding#test_removal_of_non_primary_shard","historyId":"09e31ea2bdc47f4190587a720a285af3","time":{"start":1722563303265,"stop":1722563303265,"duration":0},"description":"Tests safe removal of a shard that is not primary.","descriptionHtml":"<p>Tests safe removal of a shard that is not primary.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 842, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 874, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 696, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 531, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 616, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1090, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1722562570900,"stop":1722562570900,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1722562570901,"stop":1722562571130,"duration":229},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1722562570901,"stop":1722562570901,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1722563303266,"stop":1722563303266,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1722563303269,"stop":1722563303678,"duration":409},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1722563303679,"stop":1722563303680,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1722563303680,"stop":1722563303680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1722563303680,"stop":1722563303680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.sharding_tests"},{"name":"suite","value":"test_sharding"},{"name":"host","value":"large-441-648e1e924eaf6918646b9d8d"},{"name":"thread","value":"11158-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.sharding_tests.test_sharding"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"89c2abad7788361a","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":1,"passed":3,"unknown":0,"total":4},"items":[{"uid":"b7369ed230cbb511","reportUrl":"../2012//#testresult/b7369ed230cbb511","status":"passed","time":{"start":1722482952335,"stop":1722483847568,"duration":895233}},{"uid":"3d723260647ba0ce","reportUrl":"../2006//#testresult/3d723260647ba0ce","status":"passed","time":{"start":1722394442699,"stop":1722394995931,"duration":553232}},{"uid":"e6cbae9193161dc2","reportUrl":"../2005//#testresult/e6cbae9193161dc2","status":"passed","time":{"start":1722306636314,"stop":1722306855757,"duration":219443}}]},"tags":["runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])","asyncio","group(1)"]},"source":"26d27b889b74cde1.json","parameterValues":[]}